src/main/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/transformer/AssessmentTransformer.kt

```diff
@@ -121,7 +121,7 @@ class AssessmentTransformer(
 
     val notes = assessment.referralHistoryNotes.map {
       if (it.id == lastReferralRejectedHistoryNote?.id) {
-        assessmentReferralHistoryNoteTransformer.transformJpaToApi(it, assessment)
+        assessmentReferralHistoryNoteTransformer.transformJpaToApi(it, assessment, includeUserNotes)
```

Comment:
Yes both the notes types are notes entered by the user this why I don't renamed it. 
This can be another types in the future

Reply:
I like more includeUserNotes because it makes clear what type of notes you want to include and in this case is referring to all user notes not a specific type

Reply:
I vote for two arguments to make it clear what's happening

* includeUserNotes (existing behaviour)
* includeCustomRejectionReason (new behaviour)

And then the calling code (presumably) will control both of these based upon whether they're an assessor. This shifts the 'what to do for an assessor' logic up to the code that understands roles, and IMO makes this function's parameters easier to understand.